client/src/homepage/index.scss

```diff
@@ -4,10 +4,17 @@
   align-items: center;
   display: flex;
   flex-direction: column;
-  gap: 2rem;
   overflow-x: clip;
   overflow-y: visible;
   position: relative;
 
   width: 100%;
+
+  > * {
```

Comment:
Instead of this, could you return null here:
https://github.com/mdn/yari/blob/af6928ea1bb90f2a70d5eeb829f99d1ea87fe050/client/src/ui/organisms/placement/index.tsx#L161
To remove the section from the DOM?

Reply:
Sorry I was not precise enough. This is needed to have 0 margin in the placement is being displayed. And negative margins kill browser engineers afaik.

Reply:
Ah yes, I see what you mean